src/itemCheckbox/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component, Fragment } from 'react'
+import cc from 'classcat'
+
+import Item from '_utils/item'
+import Checkbox from '_utils/checkbox'
+import { TextDisplayType } from 'text'
+
+import style from './style'
+
+export enum ItemCheckboxStatus {
```

Comment:
Can't you use the `ItemStatus` enum? 
In every component we have the `CHECKED` status here too, so for consistency' sake, it may be better to have this here.

Reply:
There is no `CHECKED` state for checkboxes and radios where we display the green check after loading, since the change in the component's look already indicates that the value has been correctly changed.